### ISSUE TITLE
Confusing error message when removing a component without --confirm

### ISSUE THREAD

**Issue by Racer159** [zarf-dev/zarf#926]
### Environment
Device and OS: System76 popOS 22.04
App version: v0.22.1
Kubernetes distro being used: k3d
Other: N/A

### Steps to reproduce
1. Init a cluster and deploy a package
2. Run zarf package remove for that package without --confirm

### Expected result
A red message that is obvious for why it failed

### Actual Result
The help message with an error printed in white

### Visual Proof (screenshots, videos, text, etc)
![image](https://user-images.githubusercontent.com/3977569/196474097-16d31da8-73ea-46a1-9037-af2989544674.png)

### Severity/Priority
Low

### Additional Context
Also a messaging bug that could be helped by the language file work

**Comment by Madeline-UX** [comment]
@Racer159 - Language that both explains the error and then shows the correct command? I have seen something like this in a CLI before

Looks like. you are trying to remove a package.  Be sure to add -- confirm to the command.
Try this
[correct command]

**Comment by Racer159** [comment]
Yes something like that, or more highlighting of the message that exists already.  Initially I didn't notice it since it printed the help information again which was confusing initially.  In my experience commands will do this sometimes but only in explicit scenarios.  Usually if they know what you did wrong they will say only that without printing the rest of it.  Here is how apt does it:

![image](https://user-images.githubusercontent.com/3977569/199588638-20890aef-3663-43f1-99e3-6f2a862ef96f.png)

**Comment by Madeline-UX** [comment]
@Racer159 I think given that Ashton (deployer) will have limited CLI experience, adding one more line that suggest what to try instead is worth it. Thoughts?

**Comment by JoeHCQ1** [comment]
Note this is partially resolved at this point:

![image](https://github.com/user-attachments/assets/d6594322-3599-4c14-ad57-fe93a1f4d06d)

That said, as a newcomer it was still confusing that it said the "confirm flag" without clarifying "try: <the command> --confirm".

There's a bit of guesswork involved in concluding that the "confirm flag" is "--confirm".

**Comment by brandtkeller** [comment]
Closing as not planned - discussion is to evaluate more UX improvements in the future but otherwise feel the cobra defaults provide enough clarity.
